test/inductor/test_aot_inductor_utils.py

```diff
@@ -49,6 +49,15 @@ def compile(
                 restore_fqn=False,
             )
 
+        if IS_FBCODE:
+            from deeplearning.aot_inductor.extern_node_thrift_serializer import (
+                thrift_serializer,
+            )
+            if options is None:
+                options = {"extern_node_serializer": thrift_serializer}
+            else:
+                options["extern_node_serializer"] = thrift_serializer
```

Comment:
nit: you can avoid repeating yourself like so

```
if options is None:
    options = {}
options["extern_node_serializer"] = thrift_serializer
```

(yes it's slightly slower, but that battle was already lost when we started writing Python)